addons/project/static/src/components/project_right_side_panel/project_right_side_panel.xml

```diff
@@ -37,18 +37,23 @@
             </ProjectRightSidePanelSection>
             <ProjectRightSidePanelSection
                 name="'profitability'"
-                show="showProjectProfitability"
-                dataClassName="'pb-3 border-bottom o_rightpanel_full_width'"
+                show="!!this.state.data.profitability_items"
+                dataClassName="'py-3'"
             >
                 <t t-set-slot="title" owl="1">
                     Profitability
                 </t>
-                <ProjectProfitability
-                    data="state.data.profitability_items"
-                    labels="state.data.profitability_labels"
-                    formatMonetary="formatMonetary.bind(this)"
-                    onClick="(params) => this.onProjectActionClick(params)"
-                />
+                <span t-if="showProjectProfitability" class="o_default_body">
+                    <ProjectProfitability
+                        data="state.data.profitability_items"
+                        labels="state.data.profitability_labels"
+                        formatMonetary="formatMonetary.bind(this)"
+                        onClick="(params) => this.onProjectActionClick(params)"
+                    />
+                </span>
```

Comment:
Indeed, @xavierbol  I should've directly checked the condition directly in the template rather than adding an extra span. Thanks :slightly_smiling_face: